content/en/examples/application/guestbook/frontend-deployment.yaml

```diff
@@ -17,22 +18,20 @@ spec:
         tier: frontend
     spec:
       containers:
-      - name: php-redis
-        image: gcr.io/google-samples/gb-frontend:v4
+      - name: guestbook
+        image: paulczar/gb-frontend:v5
```

Comment:
This does need to change.

Reply:
There is also these [manifest changes](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/tree/abbb383b6c58e24b50b2c55fd14926e8a0b1026e/guestbook)

Reply:
it seems like the correct image would be `image: gcr.io/google-samples/gb-frontend:v5`. image version was updated here https://console.cloud.google.com/gcr/images/google-samples/GLOBAL/gb-frontend around the same time of this initial PR

Reply:
@notchairmk, OK. Is this image publically available?

Reply:
@kbhawkey yes, publicly available. Although, `gcr.io/google-samples/gb-frontend:v5` seems more tied to redis, but relies on a slightly different configuration. 

I built the guestbook app from Paul's [dockerfile PR'd against kubernetes/examples](https://github.com/kubernetes/examples/blob/d5f5aec710ef07a4396b2c241f0400fe11a8018a/guestbook/php-mongodb/Dockerfile). Unfortunately with that image the guestbook app doesn't appear to run correctly (specifically I'm currently experiencing data duplication on submit).

For both of these reasons, it might be preferable to stay with redis `gcr.io/google-samples/gb-frontend:v4` and update file/naming terminology to leader/follow.